ui/src/components/songsView/SongsTitleCard.tsx

```diff
@@ -1,30 +1,176 @@
+import InfoIcon from '@mui/icons-material/Info';
+import React, { useState } from 'react';
+import { specificSongsTabletWidth } from '../../constants';
+import SongsInfoCardMobile from './SongsInfoCardMobile';
 
 type SongTitleCardProps = {
   song: SongViewSchema | undefined;
 };
 
 const SongsTitleCard = (props: SongTitleCardProps) => {
   const song = props.song;
+  const isMobile = useMediaQuery(`(max-width:${specificSongsTabletWidth})`);
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [showMobileInfo, setShowMobileInfo] = useState(false);
+  const handlePopoverOpen = (event: React.MouseEvent<HTMLElement>) => {
+    if (!isMobile) {
+      setAnchorEl(event.currentTarget);
+    }
+  };
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleIconClick = () => {
+    if (isMobile) {
+      setShowMobileInfo(true);
+    }
+  };
+
+  const popoverOpen = Boolean(anchorEl) && !isMobile;
 
   return (
-    <>
-      <Container>
-        <Box sx={{ minWidth: 150 }}>
+    <Container>
+      <Box sx={{ minWidth: 150 }}>
+        <Box display="flex" alignItems="center">
           <Typography variant="h2" fontSize={{ sm: '24px', md: '34px' }}>
-            {song && song.title}
+            {song?.title}
           </Typography>
-          <Typography
-            variant="subtitle2"
-            color="primary.lightest"
-            fontSize={{ sm: '14px', md: '26px' }}
+          <IconButton
+            onMouseEnter={handlePopoverOpen}
+            onMouseLeave={handlePopoverClose}
+            onClick={handleIconClick}
+            sx={{ color: 'secondary.main' }}
+            aria-haspopup="true"
+            aria-owns={popoverOpen ? 'mouse-over-popover' : undefined}
           >
-            {song && song.artist}
-          </Typography>
+            <InfoIcon />
+          </IconButton>
+          {/* Desktop/tablet: hover Popover */}
+          {!isMobile && popoverOpen && (
+            <Popover
+              id="mouse-over-popover"
+              open={popoverOpen}
+              anchorEl={anchorEl}
+              onClose={handlePopoverClose}
+              anchorOrigin={{
+                vertical: 'top',
+                horizontal: 'right',
+              }}
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'left',
+              }}
+              sx={{
+                pointerEvents: 'none',
+              }}
+              slotProps={{
+                paper: {
+                  sx: {
+                    p: 2,
+                    background: '#201F25',
+                    color: '#CCC2DC',
+                    borderRadius: 2,
+                    minWidth: 140,
+                    maxWidth: 340,
+                    border: '1px solid #717171',
+                    mt: 0.5,
+                  },
+                },
+              }}
+              disableRestoreFocus
+            >
```

Comment:
you should refactor this popover into a new component for maintainability. As this is already a whole different component than the song title card.